### DISCUSSION TITLE
ValueError: Cannot deserialize model: mismatched structure

### DISCUSSION THREAD

**Discussion by lunarfeet** [explosion/spaCy#9265] [Help: Coding & Implementations] [answered]
Note: created a thread for a problem which is resolved now. Thought deleting all the text will drop the thread. Please ignore

**Comment by polm** [comment, marked as answer]
As a note to you or anyone else who sees this: Don't delete threads. Just leave a comment saying you figured it out. 

There are multiple reasons for this. 

1. It will help anyone else with the same problem.
2. If you posted it, anyone watching the Discussions gets notifications, possibly by mail. If we click through and the thread is gone it's confusing.

**Reply by lunarfeet** [reply]
@polm Noted. Will keep this in mind. Thanks
